functions/java-jre.sh

```diff
@@ -17,7 +17,32 @@ java_webupd8() {
   if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; exit 1; fi
 }
 
-java_zulu_embedded() {
+java_zulu() {
+  local FILE="/var/tmp/.zulu.$$"
+  local INSTALLROOT=/opt/jdk
+  local TEMPROOT=/opt/jdk-new
+  mkdir ${INSTALLROOT}
+  mkdir ${TEMPROOT}
+  
+  # Latest version, check https://www.azul.com/downloads/zulu-embedded
+  if is_arm; then
+    local JAVA=zulu8.33.0.134-jdk1.8.0_192-linux_aarch32hf
+  else
+    local JAVA=zulu8.33.0.1-jdk8.0.192-linux_x64
+  fi
+  whiptail --textbox $BASEDIR/includes/azul-zulu-license.md --scrolltext 27 116
```

Comment:
"docs"? Please test your code before committing changes

Reply:
I don't understand what you mean ?

Reply:
Yes it was working, pointing to docs which is where I've put the license.
I moved it to includes now, that's how I understood your comment.

Do you want me remove Oracle as part of this PR, too ?